tests/core/io/test_file_access.h

```diff
@@ -78,6 +78,26 @@ TEST_CASE("[FileAccess] CSV read") {
 	CHECK(row5[1] == "tab separated");
 	CHECK(row5[2] == "lines, good?");
 }
+
+TEST_CASE("[FileAccess] Get as UTF-8 String") {
+	Ref<FileAccess> f_lf = FileAccess::open(TestUtils::get_data_path("line_endings.lf.txt"), FileAccess::READ);
+	String s_lf = f_lf->get_as_utf8_string();
+	String s_lf_nocr = f_lf->get_as_utf8_string(true);
```

Comment:
Is `get_as_utf8_string` seeking to the start of file, or it's just trying to read from the current position (which is EOF at after previous call)?

Reply:
Oh good call, I didn't think about that.

Reply:
Seeking to 0 fixes it :+1: